.nvmrc

```diff
@@ -1 +1 @@
-10.18.1
+12.5.0
```

Comment:
I will be cautious doing that as the pipeline runs with node 10 across all the repos.
Doing this, we are possibly allowing es6 features, future code written that we won't be supported in node 10 and this could break the pipeline. I think it will be safer to keep node 10 for now.
It is worth double checking with DevOps I think